src/member.cc

```diff
@@ -1,7 +1,7 @@
 #include <assert.h>
 #include <ctype.h>
 #include <stdlib.h>
-#include <string.h>
+#include <string>
```

Comment:
Can you change the include order according to http://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes?